tools.nix

```diff
@@ -0,0 +1,15 @@
+let
```

Comment:
Why did you not call it `shell.nix`?

Reply:
`shell.nix` is already used by haskell.nix to give you ghc with all dependencies of your project. Having this file separately gives you `stack` and `nix-tools` without having to build all the dependencies